doc/source/ray-observability/user-guides/profiling.md

```diff
@@ -144,7 +144,7 @@ Find profiling results under the `/tmp/ray/session_*/logs/{profiler_name}` direc
 
 ![Nsight System Profiler folder](../images/nsight-profiler-folder.png)
 
-To visualize the results, install the [Nsight System GUI](https://developer.nvidia.com/nsight-systems/get-started#latest-Platforms) on your laptop, which becomes the host. Transfer the .nsys-rep file to your host and open it using the GUI. You can now view the visual profiling info.
+To visualize the results, install the ["Nsight Systems" 2023.3 or later GUI](https://developer.nvidia.com/nsight-systems/get-started#latest-Platforms) on your laptop, which becomes the host. Transfer the .nsys-rep file to your host and open it using the GUI. You can now view the visual profiling info.
```

Comment:
I'm not familiar enough with Nsight, but wondering if we should directly link to [this](https://developer.nvidia.com/gameworksdownload#?dn=nsight-systems-2023-3) - the currently linked page isn't great since the MacOS one points to 2023.2. This would reduce the number of steps, but the caveat is that it won't get updated if new versions are released.

Reply:
The page is currently reliable as I used Nsight System 2022.4.1 for the GUI and it can visualize `nsys-rep`. But we can add the latest version which support it.